src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -57,6 +59,12 @@ public DocumentExecuter(IDocumentBuilder documentBuilder, IDocumentValidator doc
             _documentCache = documentCache ?? throw new ArgumentNullException(nameof(documentCache));
         }
 
+        public DocumentExecuter(IDocumentBuilder documentBuilder, IDocumentValidator documentValidator, IComplexityAnalyzer complexityAnalyzer, IDocumentCache documentCache, IEnumerable<IConfigureExecution>? configurations)
```

Comment:
`IEnumerable<IConfigureExecution>?` -> `IEnumerable<IConfigureExecution>`

Reply:
Also I would like to place all initialization here - in ctor with the maximum arguments. The ctor above may delegate initizlization as it's done in first and second.

Reply:
Hard to comment at the moment, but I will be happy to review a PR.

Reply:
Then I will put all the necessary changes myself? What branch?

Reply:
If I understand what you suggested correctly, you're just suggesting a little refactoring with no effect on the public API.  Either master or develop is fine; although I'd do it in master if possible.  If there's any change in behavior, please make it its own PR and use develop if it's a breaking change.